server/src/test/java/org/elasticsearch/index/fielddata/AbstractStringFieldDataTestCase.java

```diff
@@ -445,6 +447,35 @@ public void testNestedSorting(MultiValueMode sortMode) throws IOException {
         searcher.getIndexReader().close();
     }
 
+    public void testSingleValuedGlobalOrdinals() throws Exception {
+        Document d = new Document();
+        addField(d, "_id", "1");
+        addField(d, "value", "2");
+        writer.addDocument(d);
+
+        d = new Document();
+        addField(d, "_id", "1");
```

Comment:
Should the `_id` value here be 2? or are you intentionally duplicating IDs?

Reply:
It should be 2, yeah.